README.md

```diff
@@ -1,27 +1,48 @@
-**(C) 2019 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)**<br>
-**(C) 2018 Volvo Cars**<br>
-**(C) 2016 Jaguar Land Rover**<br>
+# VEHICLE SIGNAL SPECIFICATION
 
-All files and artifacts in this repository are licensed under the
-provisions of the license provided by the LICENSE file in this repository.
 
-# VEHICLE SIGNAL SPECIFICATION
+[![License](https://img.shields.io/badge/License-MPL%202.0-blue.svg)](https://opensource.org/licenses/MPL-2.0)
+[![Build Status](https://travis-ci.org/GENIVI/vehicle_signal_specification.svg?branch=master)](https://travis-ci.org/GENIVI/vehicle_signal_specification)
+
+
+The overall goal of the Vehicle Signal Specification (VSS)  is to create a common understanding of vehicle signals in order to reach a “common language” independent of the protocol or serialisation format. 
 
 Please find the official documentation at: [Vehicle Signal Specification](https://genivi.github.io/vehicle_signal_specification/)
 
-**Cloning:**
+## Getting started
+
+### Using VSS
+To use a specific version of VSS in your toolchain, head over to our [releases page](https://genivi.github.io/vehicle_signal_specification/releases/).
+
+
+**Latest release:** [2.1](https://github.com/GENIVI/vehicle_signal_specification/releases/tag/v2.1)
+
+
+### Contribute to VSS
+Work towards version 2.2 is continuously ongoing in the [master branch](https://github.com/GENIVI/vehicle_signal_specification/tree/master)
+
+To work with the specification directly, you need to clone this repository.
 
 This repository uses git submodules.  Please make sure you clone recursively,
-or if you didn't, just init and update the submodules afterwards:
+
 ```
 git clone --recurse-submodules https://github.com/GENIVI/vehicle_signal_specification
 ```
-or
+
+
+Alternatively, just init and update the submodules after the initial cloning:
+
 ```
 git clone https://github.com/GENIVI/vehicle_signal_specification
 git submodule update --init
 ```
 
-**Latest release:** [2.1](https://github.com/GENIVI/vehicle_signal_specification/releases/tag/v2.1)
 
-Work towards version 2.2 is continuously ongoing in [master branch](https://github.com/GENIVI/vehicle_signal_specification/tree/master)
+## Contributors
+VSS is an open standard and we invite anybody to contribute. Currently VSS contains - among others - significant  contributions from
```

Comment:
I like this. Is the order intended to reflect a timeline, i.e. if someone in the future think that deserve to be listed, then they shall add their name at the end of the list.